src/main/java/org/openmainframeproject/tersedecompress/TerseDecompress.java

```diff
@@ -55,15 +63,15 @@ private void printUsageAndExit() {
 	}	
 	
     private void process (String args[]) throws Exception {
-    	
-    	String inputFileName = null;
-    	String outputFileName = null;
-    	boolean textMode = true;   	
-    	
-    	if (args.length == 0) 
+		parseArgs(args);
+    	if (args.length == 0 || (inputFileName == null && outputFileName == null) || (outputFileName == null && textMode == false) || isHelpRequested == true) 
```

Comment:
CAN BE IGNORED - check for isHelpRequested right after args.length, reasons:
- faster in case of isHelpRequested
- simpler to read code, as it first check simple things, later more complicated

Reply:
Um, this is the other dude's change I merged with. Updated